github/search.go

```diff
@@ -65,17 +65,16 @@ type CommitsSearchResult struct {
 
 // CommitResult represents a commit object as returned in commit search endpoint response.
 type CommitResult struct {
-	Hash           *string     `json:"hash,omitempty"`
-	Message        *string     `json:"message,omitempty"`
-	AuthorID       *int        `json:"author_id,omitempty"`
-	AuthorName     *string     `json:"author_name,omitempty"`
-	AuthorEmail    *string     `json:"author_email,omitempty"`
-	AuthorDate     *Timestamp  `json:"author_date,omitempty"`
-	CommitterID    *int        `json:"committer_id,omitempty"`
-	CommitterName  *string     `json:"committer_name,omitempty"`
-	CommitterEmail *string     `json:"committer_email,omitempty"`
-	CommitterDate  *Timestamp  `json:"committer_date,omitempty"`
-	Repository     *Repository `json:"repository,omitempty"`
+	SHA         *string     `json:"sha,omitempty"`
+	Commit      *Commit     `json:"commit,omitempty"`
+	Author      *User       `json:"author,omitempty"`
+	Committer   *User       `json:"committer,omitempty"`
+	Parents     []*Commit   `json:"parents,omitempty"`
+	HTMLURL     *string     `json:"html_url,omitempty"`
+	URL         *string     `json:"url,omitempty"`
+	CommentsURL *string     `json:"comments_url,omitempty"`
+	Repository  *Repository `json:"repository,omitempty"`
+	Score       *float64    `json:"score,omitempty"`
```

Comment:
As an observation, `CommitResult` now looks very similar to [RepositoryCommit](https://godoc.org/github.com/google/go-github/github#RepositoryCommit). But it differs in the last 2 fields. `RepositoryCommit` has `Stats` and `Files`, which `CommitResult` does not, and `CommitResult` has `Repository` and `Score`, that `RepositoryCommit` doesn't.

That's completely fine, just pointing out.

However, I think it'd be nice to add a blank line between `CommentsURL` field and `Repository` field, to separate the common vs differing fields, similar to the blank line separator that `RepositoryCommit` already has.